config/tizen/packaging/iotjs.spec

```diff
@@ -0,0 +1,105 @@
+Name: iotjs
+Version: 1.0.0
+Release: 0
+Summary: Platform for Internet of Things with JavaScript
+Group: Network & Connectivity
+License: Apache-2.0
+URL: https://www.iotjs.net/
+Source:     %{name}-%{version}.tar.gz
+Source1:    %{name}.pc.in
+Source1001: %{name}.manifest
+
+BuildRequires: python
+BuildRequires: cmake
+BuildRequires: glibc-static
+BuildRequires: aul
+BuildRequires: pkgconfig(appcore-agent)
+BuildRequires: pkgconfig(capi-appfw-service-application)
+BuildRequires: pkgconfig(capi-appfw-app-common)
+BuildRequires: pkgconfig(capi-appfw-package-manager)
+BuildRequires: pkgconfig(capi-appfw-application)
+BuildRequires: pkgconfig(dlog)
+#for https
+BuildRequires:  openssl-devel
+BuildRequires:  libcurl-devel
+
+Requires(postun): /sbin/ldconfig
+Requires(post): /sbin/ldconfig
+
+%description
+Platform for Internet of Things with JavaScript
+
+# default is RELEASE mode.
+# If DEBUG mode is needed, please use tizen_build_devel_mode
+%define RELEASE False
+# For Example
+%if %{RELEASE} == "True"
+%define build_mode release
+%else
+%define build_mode debug
+%endif
+
+# Default values to be eventually overiden BEFORE or as gbs params:
+%{!?RELEASE: %define RELEASE 0}
+
+%package service
+Summary: Development files for %{name}
+Group: Network & Connectivity/Service
+Requires: %{name} = %{version}-%{release}
+
+%description service
+The %{name}-service package contains service iotjs files for
+developing applications that use %{name}.
+
+%package devel
+Summary: Header files for %{name}
+Group: Network & Connectivity/Service
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+Development libraries for %{name}
+
+%prep
+%setup -q -c
+chmod g-w %_sourcedir/*
+cat LICENSE
+cp %{SOURCE1001} .
+touch abc.txt
+
+%build
+./tools/build.py --clean --buildtype=%{build_mode} --target-arch=arm \
+ --target-os=tizen --target-board=artik10 \
+ --no-init-submodule --no-parallel-build --no-check-test
```

Comment:
Tizen profiles runs on Travis still, not related with GBS. (I changed the code on the review)
It's possible to make Tizen profile travis to create a .rpm. However, It's possible to guarantee as the Tizen OBS system and a maintainer apply iot.js source code on the Tizen.org. So I didn't make it.

For the dependency, let me explain more.
1. The files in './conf/tizen/packaing' are for the GBS building, which runs on the developer's PC and Tizen OBS system to create RPM Package.
2. A file, 'gbsbuild.sh' is only for developer who wants make own IoT.js module or changes on Tizen device.